master/internal/command/tensorboard_manager.go

```diff
@@ -43,7 +43,7 @@ const (
 
 // TensorboardRequest describes a request for a new Tensorboard.
 type TensorboardRequest struct {
-	CommandParams
+	*CommandParams
```

Comment:
Go style nit: I prefer struct embedding only when it embeds behavior not fields, though I'm not sure what the standard is here though. (plus we don't even use anything gained from embedding it).

Reply:
would the aleternative be to redefine every attribute in commandparams or maybe nest it? the former wouldn't be as DRY

Reply:
It would be like
```suggestion
	CommandParams *CommandParams
```
See https://golang.org/doc/effective_go.html#embedding, in the code it's embedding all the methods and fields of `CommandParams` and they're accessible as if `TensorboardRequest` inherits them.

Reply:
sg I had updated the shells like this https://github.com/determined-ai/determined/pull/1454/commits/d3f123e5ae237abb8ead30015b87af098ee194d8 I'll make the rest the same.